src/plugins/remove/azureRemovePlugin.ts

```diff
@@ -40,10 +41,25 @@ export class AzureRemovePlugin extends AzureBasePlugin {
   }
 
   private async remove() {
-    const resourceClient = new ResourceService(this.serverless, this.options);
-    await resourceClient.deleteDeployment();
-    await resourceClient.deleteResourceGroup();
-
-    this.log("Service successfully removed");
+    const resourceService = new ResourceService(this.serverless, this.options);
+    const rg = await resourceService.getResourceGroup();
+    const rgName = resourceService.getResourceGroupName();
+    if (!rg) {
+      this.log(`Resource group "${rgName}" does not exist in your Azure subscription`)
+      return;
+    }
+    this.log(`This command will delete your entire resource group (${resourceService.getResourceGroupName()}). ` +
```

Comment:
nit: can we update the message to indicate something like: "This command will delete your ENTIRE resource group <rg name> and ALL the Azure resources that it contains."

Just to explicitly call out that all resources will get nuked, not just things created by the sls framework.